modules/update-conditional-web-console.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-web-console.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-conditional-web-console{context}"]
+= Performing conditional updates in the web console
+
+You can view and assess the risks associated with particular updates with conditional updates.
+
+.Prerequisites
+* You have access to the cluster with `cluster-admin` privileges.
+
+* You have access to the OpenShift Container Platform web console.
+
+* Pause all MachineHealthCheck resources.
+
+* Your Operators previously installed through Operator Lifecycle Manager (OLM) are updated to their latest version in their latest channel. Updating the Operators ensures they have a valid update path when the default OperatorHub catalogs switch from the current minor version to the next during a cluster update.
+
+* Your machine config pools (MCPs) are running and not paused. Nodes associated with a paused MCP are skipped during the update process. You can pause the MCPs if you are performing an advanced update strategy, such as a canary rollout, an EUS update, or a control-plane update.
+
+.Procedure
+
+. From the web console, click *Administration* -> *Cluster settings* page and review the contents of the *Details* tab.
+
+. Expand the <need to figure out what UI element is called> to display the potential risks to updating.
```

Comment:
Maybe notification?

Reply:
Suggested text: expand the `select new version` UI element

Reply:
I am fine with any-better wording as well.